_posts/2019-12-09-migrations-meetup.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title: "Lightning talks with women of Scribd: migrations, owning quality, and more!"
+author: harinii
+tags:
+- meetup
+- toronto
+- featured
+team: Payments
+---
+
+As we build [the largest library in the world](/blog/2019/building-the-library.html), we need more manpower, and more
+importantly more **womanpower**. As we make plans to scale our product and make way
+to scale our users, we need a way to scale our engineering teams as well. We
+are actively looking to expand our teams at all our offices and particularly in
+Toronto. Between the University of Toronto and Waterloo, we discovered long ago
+that there are lots of talented people around Toronto, and have since made
+Toronto the home of our second engineering office! Partnering with Women Who
+Code Toronto chapter seemed like a perfect way for us to launch Scribd's
+footprint as an engineering brand and attract a diverse talent.  Four women
+from Scribd engineering flew to Toronto from San Francisco last week to talk
+about a wide range of topics. The
+[event](https://www.meetup.com/Women-Who-Code-Toronto/events/266758297/) was
+organized by our SF based recruiter Lucas and our HR partner in Toronto office,
+Tammy who took care of every detail for the event to be successful. There were
+about 50+ attendees to the event, mostly women. We event started off with
+Katerina welcoming the
+guests and giving a brief overview of what we're building at Scribd, and then
+went straight into the talks!
+
+
+First up  was **Paige Stone** (Software Engineer, Notifications) who presented "Green Field projects
+and how I learnt to love Migrations."  She talked about the importance of
+evaluating and determining whether solution involves re-iterating through
```

Comment:
How about "solution should be to re-iterate through existing code or build something from scratch"